emcc.py

```diff
@@ -2860,8 +2860,7 @@ def do_binaryen(target, options, wasm_target):
     building.eval_ctors(final_js, wasm_target, debug_info=intermediate_debug_info)
 
   # after generating the wasm, do some final operations
-  if shared.Settings.SIDE_MODULE:
-    webassembly.add_dylink_section(wasm_target, shared.Settings.RUNTIME_LINKED_LIBS)
+  webassembly.add_dylink_section(wasm_target, shared.Settings.RUNTIME_LINKED_LIBS)
```

Comment:
shouldn't this check for main or side modules, as we don't want it otherwise?

Reply:
Oh course, doh!